libcontainer/container_linux.go

```diff
@@ -836,6 +836,80 @@ func (c *linuxContainer) handleCriuConfigurationFile(rpcOpts *criurpc.CriuOpts)
 	}
 }
 
+func (c *linuxContainer) handleCheckpointingExternalNamespaces(rpcOpts *criurpc.CriuOpts, minVersion int, t configs.NamespaceType) error {
```

Comment:
(I swear I left a comment about `minVersion` but I can't see it. Hmm, did I do it in my sleep?)

I think the `minVersion` should not be specified by the caller but internalized in this method, as it is clearly a function of NamespaceType.

IOW

```go
func (c *linuxContainer) handleCheckpointingExternalNamespaces(rpcOpts *criurpc.CriuOpts,t configs.NamespaceType) error {
var minVersion int
if t == configs.NEWNET {
￼		nsType = "Net"
￼		minVersion = 31100
}
```
￼	} else {